answers.js

```diff
@@ -4,12 +4,329 @@ Write a program that will generate a number between 1 and 100. Your program will
 function(randomno) {
   var compuguess = computer guess
   if (guess === compuguess){
-    return 
+    return
   }
 else if (guess > compuguess || guess < compuguess) {
```

Comment:
You'd likely want to have different conditional paths/two separate `else if` statements for if `guess > computerGuess` and if `guess < computerGuess`, because the program should give two different responses for each one.